vlm/src/main/scala/geotrellis/contrib/vlm/effect/geotiff/BaseGeoTiffRasterSource.scala

```diff
@@ -0,0 +1,31 @@
+package geotrellis.contrib.vlm.effect.geotiff
+
+import geotrellis.contrib.vlm.effect.RasterSourceF
+import geotrellis.contrib.vlm.geotiff.{GeoTiffMetadata, GeoTiffPath}
+import geotrellis.raster.CellType
+import geotrellis.raster.io.geotiff.reader.GeoTiffReader
+import geotrellis.raster.io.geotiff.{MultibandGeoTiff, Tags}
+import geotrellis.util.RangeReader
+
+import cats.Monad
+import cats.syntax.functor._
+import cats.syntax.apply._
+
+abstract class BaseGeoTiffRasterSource[F[_]: Monad: UnsafeLift] extends RasterSourceF[F] {
```

Comment:
I think we should avoid `Base_` raster sources. It doesn't really win much and adds a layer of indirection. I think in this case just duplicating those method implementation is perfectly reasonable.